src/Orchestration/Adapter/DockerCLI.php

```diff
@@ -372,6 +370,9 @@ public function run(string $image,
             throw new Orchestration("Docker Error: {$output}");
         }
 
+        // Use first line only, CLI can add warnings or other messages
+        $output = \explode("\n", $output)[0];
```

Comment:
Fixes running locally, warning was being added to container ID and breakingn some tests